spring-boot-admin-server/src/main/java/de/codecentric/boot/admin/server/config/AdminServerInstanceWebClientConfiguration.java

```diff
@@ -214,6 +214,12 @@ public LegacyEndpointConverter mappingsLegacyEndpointConverter() {
 			return LegacyEndpointConverters.mappings();
 		}
 
+		@Bean
```

Comment:
I did not yet take a look what this bean is for, but it seems the travis build failed due to a test failure in `AdminServerInstanceWebClientConfigurationTest` where this new bean `startupLegacyEndpointConverter` is not yet expected.